src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java

```diff
@@ -40,7 +41,7 @@ public class EncounterPerformed extends DataElement {
   private Interval relevantPeriod;
   private Code dischargeDisposition;
   private List<Code> facilityLocations;
-  private List<Code> diagnoses;
+  private List<Component> diagnoses;
```

Comment:
I think this was always incorrect. Correct attribute would be DiagnosisComponent
```suggestion
  private List<DiagnosisComponent> diagnoses;
```
![image](https://github.com/user-attachments/assets/a1765222-5886-4e69-8506-7bd10b0ec0cf)

Reply:
@adongare Yes you are right, this is the data: 
![image](https://github.com/user-attachments/assets/cd2eabb9-7d98-4dd3-b928-0d67c16d541e)
I used a more generic class and it seems working fine: 
![image](https://github.com/user-attachments/assets/5217f9d1-4322-4e73-a42e-b01052369ffb)
Let me know if you think we should create a new class for DiagnosisComponent, thanks Ashok!

Reply:
yeah, that is incorrect. diagnosis should have these fields.
<img width="983" alt="image" src="https://github.com/user-attachments/assets/b35a2ac9-6253-4bd5-9668-9bccee558ace">

Reply:
and if in case we did not implement support for this in UI, we'll have to enter the tickets

Reply:
@adongare Ok thanks so much for the feedback, I'll create a new class for DiagnosisComponent.

Reply:
@adongare I created and used DiagnosisComponent for EncounterPerformed. Thanks!